engine/src/test/java/org/operaton/bpm/engine/test/api/mgmt/ManagementServiceAsyncOperationsTest.java

```diff
@@ -576,7 +576,7 @@ public void shouldSetInvocationsPerBatchTypeForJobsByProcessInstanceIds() {
   }
 
   protected List<String> getAllJobIds() {
-    return getAllJobs().stream().map(Job::getId).collect(Collectors.toList());
+    return getAllJobs().stream().map(Job::getId).toList();
```

Comment:
Stream#toList produces an unmodifiable list, making a test setup which adds IDs later complicated (or in this case, breaks), see line 189